test/e2e/Makefile

```diff
@@ -74,15 +79,20 @@ ginkgo: # Make sure ginkgo is in $GOPATH/bin
 
 .PHONY: run
 run: ginkgo
+		@[ "${UPGRADE_FROM_VELERO_CLI}" ] && echo "Upgrade from ${UPGRADE_FROM_VELERO_CLI}" || \
```

Comment:
Move the checking into the upgrade test case, this prevents running e2e even though I don't want to cover the upgrade case

Reply:
It's moved into upgrade script.